control/control.go

```diff
@@ -814,7 +848,7 @@ func (p *pluginControl) FetchMetrics(ns []string, version int) ([]core.Cataloged
 }
 
 func (p *pluginControl) GetMetric(ns []string, ver int) (core.CatalogedMetric, error) {
-	return p.metricCatalog.Get(ns, ver)
+	return p.metricCatalog.GetMetric(ns, ver)
 }
 
 func (p *pluginControl) GetMetricVersions(ns []string) ([]core.CatalogedMetric, error) {
```

Comment:
I think this stems from the desire to "get" metrics using a wildcard through REST (i.e. `snapctl metric get -m "/intel/psutil/load/*" -v 5`).  @IRCody, is this correct?

Assuming that this is related to adding the ability to get list/get metrics using wild cards through REST we should consider adding that as a separate issue/PR (feature).